packages/luna/src/locales/ar-eg.json

```diff
@@ -402,5 +402,8 @@
   "error.schedule.duration": "أقل مده هي 30 دقيقة",
   "error.tutors.list.error": "حدثت مشكلة أثناء تحصيل قائمة المعلمين، الرجاء المحاولة لاحقاً",
   "error.tutors.list.empty": "يبدو أنه لا يوجد معلمين متاحين الان، الرجاء المحاولة لاحقاً",
-  "error.unexpected": "حدث خطاء غير متوقع، الرجاء المحاولة لاحقاً"
+  "error.unexpected": "حدث خطاء غير متوقع، الرجاء المحاولة لاحقاً",
+  "tutor.rate.succes": "تم تقييم المعلم بنجاح",
+  "tutor.rate.error": "حدث خطأ أثناء تقييم المعلم، برجاء المحاولة مرة أخرى",
+  "tutor.rate.editmessage": "تعديل تقييم المعلم"
```

Comment:
Please move these message at the top (may be below this [line](https://github.com/litespace-org/litespace/blob/5723ee988380ed9397871cafaeae9bebbc66d0bf/packages/luna/src/locales/ar-eg.json#L34)). The end of the file is reserved for error labels.